content/docs/sidebar.json

```diff
@@ -91,6 +91,26 @@
         "label": "What is DVC?",
         "slug": "what-is-dvc"
       },
+      {
+        "label": "Basic Concepts",
+        "slug": "concepts",
+        "source": false,
+        "children": [
+          "workspace",
+          "dvc-project",
+          "dependency",
+          "external-dependency",
+          "data-pipelines",
+          "stage",
+          "dvc-cache",
+          "dvc-metafiles",
+          "remote-storage",
+          "import-stage",
+          "metrics-and-plots",
+          "parameters",
+          "run-cache"
+        ]
+      },
```

Comment:
Too many changes in a single PR 🙂  Let's start with the index (if there's one) and 1-3 of the most important concept pages?

That said, I also have a couple comments on (1) the concept naming, and (2) level of abstraction:

1. Let's use `dvc-` everywhere (where it can apply) or nowhere, I think.
2. Not everything that has a tooltip needs a concept page e.g. "external dependency" is a bit specific and falls within the "dependency" concept. Not sure which ones to keep exactly, please suggest.

Thanks

Reply:
Also on the level of abstraction, we could even consider putting related concepts together e.g. cache and remote, pipeline and stage (they could still each need separate tooltips though). WDYT?

Reply:
p.s. please read my comments on Guthub because I've edited some from the original text you get via email.

Reply:
1. I agree with this. I tried to make minimal changes to file naming as I think that was a decided matter but we can change all to non-`dvc` versions IMO. 
2. You're also right on this but tooltips in the branch is using frontmatter and there can only be a single frontmatter for each page. We can either have a _dependency_ tooltip/glossary and tell external dependency when the user visits the concept page, or all glosses should have separate pages. IMHO **glossary/tooltips and concepts should be separated** and we need to have 5-10 concept pages and put tooltips/glossary in a single file. 
3. This also needs separation of glossary and basic concepts. Current structure requires single page, short documents for concepts which has tooltips defined at the top.

Reply:
On 1. We're moving all the files to another directory I think so no problem in renaming them along the way, I think.
On 2. I think we can have empty concept md docs that only serve for tooltip purposes, but not link them in sidebar.js right?
On 3. True. For now we can pick which concept doc is more important and put other concepts in there (even if they have their own concept file that's only used for tooltips. Anyway, no need to worry about this for the first or 2nd PR probably...

Reply:
p.s. the tooltip vs concept page separation part we can worry about when looking into #1395